moveit_ros/planning/planning_request_adapters_plugin_description.xml

```diff
@@ -30,4 +30,9 @@
     </description>
   </class>
 
+  <class name="default_planner_request_adapters/AddIterativeSplineParameterization" type="default_planner_request_adapters::AddIterativeSplineParameterization" base_class_type="planning_request_adapter::PlanningRequestAdapter">
```

Comment:
I think this should be:
```
<class name="default_planner_request_adapters/AddSplineParameterization" type="default_planner_request_adapters::AddSplineParameterization" base_class_type="planning_request_adapter::PlanningRequestAdapter">
    <description>
    </description>
  </class>
```
To match the names in `add_iterative_spline_parameterization.cpp`